packages/shared/src/messages/MessageProxy.ts

```diff
@@ -124,6 +124,7 @@ export class RpcBrowser {
   counter: number = 0;
   promises: Map<number, { resolve: (value: unknown) => unknown; reject: (value: unknown) => void }> = new Map();
   subscribers: Map<string, (msg: any) => void> = new Map();
+  noTimeoutChannels: string[] = ['openDialog'];
```

Comment:
please make it available somewhere like the Messages ids in consts file or something

Reply:
Done